local_store.go

```diff
@@ -431,11 +431,9 @@ func (f *fileSystemStore) Commit(consistentSnapshot bool, versions map[string]in
 			return err
 		}
 		if !info.IsDir() && isTarget(rel) && needsRemoval(rel) {
-			//lint:ignore SA9003 empty branch
 			if err := os.Remove(path); err != nil {
-				// TODO: log / handle error
+				return err
```

Comment:
We were ignoring file removal failures and now, with this change, we are not? That seems like a reasonable change, are there likely to be any unexpected consequences of this change?

Reply:
Good question. Maybe we should add a test case, if it's missing?

Reply:
I tested several use cases and I didn't find a reason to break something. It's good to leave this check though and address it again in case someone raises a valid reason to change it back 👍